app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,331 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/colorShadowBackground"
+    tools:context=".MovieDetailsActivity">
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        tools:context=".MovieDetailsActivity">
+
+        <androidx.appcompat.widget.AppCompatImageView
+            android:id="@+id/movie_poster"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layerType="software"
+            android:scaleType="centerCrop"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:srcCompat="@drawable/avengers_poster" />
+
+
+        <androidx.appcompat.widget.AppCompatImageView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:scaleType="fitXY"
+            android:contentDescription="@string/movie_poster_content_description"
+            app:layout_constraintBottom_toBottomOf="@id/movie_poster"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="@id/movie_poster"
```

Comment:
In order to fix the corrupted emulator image I would try also to create a View with a background. Have you tried this?
Something like this (but with correct constraints) : 
```
        <View
            android:id="@+id/darkModeOverlay"
            android:layout_width="match_parent"
            android:layout_height="match_parent"
            android:background="@drawable/shadow_bottom" />
```

Reply:
I tried this but the result was the same.
```
<View
    android:layout_width="match_parent"
    android:layout_height="wrap_content"
    app:layout_constraintBottom_toBottomOf="@id/movie_poster"
    app:layout_constraintEnd_toEndOf="parent"
    app:layout_constraintStart_toStartOf="parent"
    app:layout_constraintTop_toTopOf="@id/movie_poster"
    android:background="@drawable/shadow_bottom" />
```

Reply:
> In order to fix the corrupted emulator image I would try also to create a View with a background. Have you tried this?
> Something like this (but with correct constraints) :
> 
> ```
>         <View
>             android:id="@+id/darkModeOverlay"
>             android:layout_width="match_parent"
>             android:layout_height="match_parent"
>             android:background="@drawable/shadow_bottom" />
> ```

I tried this but the result was the same.
```
<View
    android:layout_width="match_parent"
    android:layout_height="wrap_content"
    app:layout_constraintBottom_toBottomOf="@id/movie_poster"
    app:layout_constraintEnd_toEndOf="parent"
    app:layout_constraintStart_toStartOf="parent"
    app:layout_constraintTop_toTopOf="@id/movie_poster"
    android:background="@drawable/shadow_bottom" />
```

Reply:
You could also try using png instead of xml gradient 🤔 
Like this one: https://raw.githubusercontent.com/achirkof/MovieApp/e90dbb4fea0c69ddaf39b787a123a908426e474b/app/src/main/res/drawable/img_cover_mask.png

I know it's not a "perfect solution", but still